README.md

```diff
@@ -130,7 +130,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [IPInfo.io](https://ipinfo.io/) - The trusted source for IP address data
 - [IPData.co](https://ipdata.co/) - IP Geolocation and Threat Intelligence API
 - [NetworksDB](https://networksdb.io/) - information about the public IPv4 and IPv6 addresses, networks and domains owned by companies and organisations across the world
-- [ASNlookup](https://asnlookup.com/) - Quickly lookup updated information about specific Autonomous System Number (ASN), Organization, CIDR, or registered IP addresses (IPv4 and IPv6) among other relevant data
+- [ASNlookup](https://asnlookup.com/) - Quickly lookup updated information about specific Autonomous System Number (ASN), Organisations, CIDR, or registered IP addresses (IPv4 and IPv6) among other relevant data
```

Comment:
If you google `asnlookup com`, you'll see its official description. it's not a typo